packages/api/server/channels/app.mts

```diff
@@ -167,10 +172,54 @@ async function dependenciesInstall(
         status: code === 0 ? 'complete' : 'failed',
         code,
       });
+
+      if (code === 0) {
```

Comment:
In terms of the UI. Does sending this event earlier make a difference with a flash state?